app/main.py

```diff
@@ -1,23 +1,38 @@
 from enum import Enum
 
 from bokeh.embed import server_document
+from dotenv import load_dotenv
```

Comment:
Following the PEP8 guidelines, imports should be grouped in:
1. Standard imports
2. Third party libraries
3. Our own modules

So `dotenv` and `os` should be up in the first group.

Reply:
dotenv is an extra package, but `os` is at the wrong place indeed!

Reply:
Sorry, I realized `dotenv` is not a standard library, then it is ok here